### PR TITLE
[14.0][MIG] partner_company_type

### PR DESCRIPTION
**Pull request by kittiu**
Standard migration, ready for review.
(almost nothing is changed).

### CODE REVIEW

**Review by tonygalmiche — APPROVED** [review]
LGTM : Code review and functional review with runbot

image

**Review by HaraldPanten — APPROVED** [review]
Functional review

File: partner_company_type/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* Denis Roussel <denis.roussel@acsone.eu> (https://acsone.eu)
+* Gilles Meyomesse <gilles.meyomesse@acsone.eu> (https://acsone.eu)
```
Comment (newtratip) [inline partner_company_type/readme/CONTRIBUTORS.rst:2]: Please add your name to contributers.

Comment (kittiu) [reply partner_company_type/readme/CONTRIBUTORS.rst:2]: It is ok. I almost did nothing to migrate.

**Comment by newtratip** [comment]
Code reviewed ok

**Review by newtratip — APPROVED** [review]
Code reviewed ok.

**Comment by tonygalmiche** [comment]
@jgrandguillaume @dreispt @pedrobaeza I think it is possible to merge. Thank you in advance.

**Review by bodedra — APPROVED** [review]

**Pull request merged**